### DISCUSSION TITLE
CRAN submission error for checking a package built using instantiate

### DISCUSSION THREAD

**Discussion by xinxinc476** [wlandau/instantiate#20] [Q&A] [answered]
I'm working on building an R package using `instantiate`.  The [package](https://github.com/ethan-alt/hdbayes) passes all R CMD checks on my local machine without any errors. However, when submitted to CRAN, it generates the following error:
```
Error:
! in callr subprocess.
Caused by error:
! The package "cmdstanr" is required The {cmdstanr} package is required in
order to install CmdStan and run Stan models. Please install it manually using
install.packages(pkgs = "cmdstanr", repos =
c(https://mc-stan.org/r-packages/, getOption("repos"))
```
It looks like this error occurs because `cmdstanr` is not available on CRAN and needs to be installed. 
I would greatly appreciate any suggestions for resolving this error.

**Comment by wlandau** [comment, marked as answer]
Sorry about that. Should be fixed in 0a2111a6fce8b6b6bea6607030ee5e1f7357733a.

**Reply by xinxinc476** [reply]
Thanks a lot for the update! I have used the latest version of `instantiate` package for my own package and checked my package using `rhub::check_for_cran()`. The package did not generate any error from R-hub when using the platform Debian Linux, R-devel, GCC ASAN/UBSAN. However, it results in the following error when using the platform **Windows Server 2022, R-devel, 64 bit**:

`Package required and available but unsuitable version: 'instantiate'`

I was wondering if you could help fix this issue.

**Reply by wlandau** [reply]
R-hub might be using an outdated version of `instantiate` because I only submitted the update last week. You can check the R-hub logs to confirm, but it sounds like your DESCRIPTION file requires a minimum version of `instantiate` greater than the version currently installed on R-hub.
